### PR TITLE
Enable Logs page

### PR DESCRIPTION
**Pull request by mickaelandrieu**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Search feature was validated previously, I just forgot to enable again Logs page as we were improving the Grid system. Now it's time to migrate it for good, don't you think?
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Access Configure > Advanced Parameters > Logs

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9457)

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
I see that you drop several files about Logs, but you keep the controller?

**Comment by sarjon** [comment]
@mickaelandrieu you can remove `FilterLogsByAttributeType` and rely on form that is built by grid.

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by mickaelandrieu** [comment]
@sarjon done! We really need to update docs :/ there is too much things to remember for my goldfish brain !

**Review by Quetzacoalt91 — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/grid_actions.html.twig [hunk 1]
```diff
@@ -22,13 +22,12 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-
 {% if grid.actions.grid|length > 0 %}
-  {% set girdActionsButtonId = grid.id-grid ~ '-actions-button' %}
+  {% set gridActionsButtonId = grid.id ~ '-grid-actions-button' %}
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/grid_actions.html.twig:26]: funny thing: I'll be able to detect this issue on Survival tests *only*. In dev mode, we should [be allowed](https://symfony.com/doc/current/reference/configuration/twig.html#strict-variables) to ask Twig to be really strict...

**Comment by mickaelandrieu** [comment]
ping @Quetzacoalt91, this needs an extra review as I've found and fixed an error when I've uncommented my "survival" test :muscle:

File: tests/Integration/PrestaShopBundle/Test/WebTestCase.php [hunk 2]
```diff
@@ -190,12 +199,14 @@ protected function enableDemoMode()
...
-            array('_PS_MODULE_DIR_', null, __DIR__.'/../../../resources/modules/'),
+            array('_PS_MODULE_DIR_', null, __DIR__ . '/../../../resources/modules/'),
         );
 
         $configurationMock->method('get')
             ->will(self::returnValueMap($values));
 
         self::$kernel->getContainer()->set('prestashop.adapter.legacy.configuration', $configurationMock);
     }
+
```
Comment (PierreRambaud) [inline tests/Integration/PrestaShopBundle/Test/WebTestCase.php]: Useless EOL

**Review by PierreRambaud — DISMISSED** [review]
Useless EOL but :+1:

File: tests/Integration/PrestaShopBundle/Test/WebTestCase.php (hunk 2 above)
Comment (mickaelandrieu) [reply tests/Integration/PrestaShopBundle/Test/WebTestCase.php]: done

**Comment by mickaelandrieu** [comment]
I can't explain to myself why this is not working in PHP 5.6, any idea?

**Review by PierreRambaud — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
@mickaelandrieu I will try to have a check on it ;)

**Comment by PierreRambaud** [comment]
@mickaelandrieu 
```
Uncaught PHP Exception ReflectionException: "Class PrestaShop\PrestaShop\Core\Search\Filters\LogsFilters does not exist" at /home/prambaud/projects/prestashop/source/develop/vendor/sensio/framework-extra-bundle/EventListener/ParamConverterListener.php line 84 
```

**Comment by PierreRambaud** [comment]
Hey @mickaelandrieu There is a notice:
> Static function PrestaShop\PrestaShop\Core\Search\Filters::getDefaults() should not be abstract

 That's why test failed ;)

**Comment by Quetzacoalt91** [comment]
@mickaelandrieu 

The error is the following:
```
PHPUnit_Framework_Error_Notice

Static function PrestaShop\PrestaShop\Core\Search\Filters::getDefaults() should not be abstract
```

It then throws the undefined class LogsFilter.

It seems PHP does not allow abstract static methods on strict mode. I have to replace the Filters::getDefaults method with the following:

```php
    /**
     * @return array Define the default filters configuration
     */
    public static function getDefaults()
    {
        return [];
    }
```

**Comment by mickaelandrieu** [comment]
Wow: sneaky... well this is a proof of how cool is it to have - at least - survival tests: easy to write and so many gains (and so much pain :p) already!

Thanks for the stack trace :+1: 
cool

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @mickaelandrieu ;)

**Pull request merged**